frontend/actions.js

```diff
@@ -95,23 +95,30 @@ export default ({setState, getState}) => {
     }
   }
 
-  const validateSendAddress = (state) => setState({
-    sendAddress: Object.assign({}, state.sendAddress,
-      {validationError: !Cardano.isValidAddress(state.sendAddress.fieldValue) ? {code: 'SendAddressInvalidAddress'} : ''}),
-  })
+  const validateSendAddress = (state) =>
+    setState({
+      sendAddress: Object.assign({}, state.sendAddress, {
+        validationError: !Cardano.isValidAddress(state.sendAddress.fieldValue)
+          ? {code: 'SendAddressInvalidAddress'}
+          : '',
+      }),
+    })
 
   const validateSendAmount = (state) => {
     let validationError = ''
+    const strAmount = state.sendAmount.fieldValue
     const amount = parseFloat(state.sendAmount.fieldValue)
+
     if (isNaN(amount)) {
       validationError = {code: 'SendAmountIsNan'}
     }
-    if (parseFloat(amount.toFixed(6)) <= 0) {
-      validationError = {code: 'SenAmountIsNotPositive'}
+    if (strAmount.split('.').length === 2 && strAmount.split('.')[1].length > 6) {
```

Comment:
I am not sure if there is another validation somewhere else but a user can trick this by using
`1.0e-8` as as `strAmount`

Reply:
Yes, you are right, i forgot about this case. I added a check for the letter "e" above, so any scientific notation would be rejected as an invalid input.